### DISCUSSION TITLE
Zotero integration not working on Windows 10

### DISCUSSION THREAD

**Discussion by asbrandt** [BryanJenksCommunity/FAQ#177] [Q&A]
Hi All,

I followed the workflows by Bryan and by Danny Hatcher, and it's working perfectly on my mac but not on my Windows computer (provided by work). Because I get better access to papers/research tools on my work computer, I'd love to be able to use this on Windows. When I run zotero integration, I get the zotero search bar, can search for the paper whose notes I want to import, the green bar goes along like it is going to work, but no note is created. Anyone have any ideas? 

Thank you!

**Comment by tallguyjenks** [comment]
what do your plugin settings look like?
